types/validator_set.go

```diff
@@ -626,10 +624,11 @@ func (vals *ValidatorSet) VerifyCommit(chainID string, blockID BlockID, height i
 		// }
 	}
 
-	if talliedVotingPower > vals.TotalVotingPower()*2/3 {
-		return nil
+	if got, needed := float32(talliedVotingPower), float32(vals.TotalVotingPower()*2/3); got <= needed {
```

Comment:
`float32` conversion seems like a recipe for disaster. Can we multiply by integers instead?

Reply:
... especially in a chain through IBC, I don't think we can do this, since it might be machine-dependent.